crates/ibc/src/core/ics24_host/identifier/validate.rs

```diff
@@ -1,19 +1,12 @@
 use super::IdentifierError as Error;
 use crate::prelude::*;
 
-/// Path separator (ie. forward slash '/')
-const PATH_SEPARATOR: char = '/';
```

Comment:
Don't remove this. I noticed that the source code uses `'/'` in many places. We should use this instead.

Reply:
It should be added (in another commit) to and used in ibc/src/core/ics24_host/path.rs then.  If I leave it here, it’s an unused identifier:

    error: constant `PATH_SEPARATOR` is never used

Reply:
I see. It's okay for now. I will create a separate issue for this.